docs/content/wiring.md

```diff
@@ -25,6 +25,35 @@ You can also provide any other attributes, as specified by the CNAB [parameters]
   default: "wordpress"
 ```
 
+### File Parameters
+
+Porter also enables the use of file parameters in a bundle.
+
+For instance, a bundle might declare a parameter `mytar` of type `file`, to exist at `/root/mytar` in the runtime container:
+
+```yaml
+- name: mytar
+  type: file
+  path: /root/mytar
+```
+
+which can be used in a step like `install`:
+
+```yaml
+install:
+  - exec:
+      description: "Install"
+      command: bash
+      flags:
+        c: tar zxvf /root/mytar
+```
+
+Passing in the local file representing the `mytar` parameter is done similarly to any other parameter:
```

Comment:
Suggestion: "The syntax to pass a parameter to `porter` is the same for both regular and file parameters:"